dotfiles_setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+printf "\n\n$(tput bold)$(tput setaf 7)$(tput setab 2)Creating CFPB standard dotfiles$(tput sgr 0)"
```

Comment:
can you put a newline between  the message here.  I see this:
<img width="848" alt="Screen Shot 2021-06-29 at 12 36 10 PM" src="https://user-images.githubusercontent.com/1038306/123835488-99cecd00-d8d6-11eb-9cc4-c10abe1951ea.png">

Reply:
Done in 82c2ad1